src/bidiMapper/domains/log/logHelper.ts

```diff
@@ -52,13 +52,21 @@ export function logMessageFormatter(
       if (token === '%s') {
         output += stringFromArg(arg);
       } else if (token === '%d' || token === '%i') {
-        if (['bigint', 'number', 'string'].includes(arg.type)) {
+        if (
```

Comment:
I am not sure why but hand-written types are slightly different and TS could infer `includes` properly.

Reply:
ouch )